### PR TITLE
K8s 1.25 + More Improvements

### PR DESCRIPTION
**Pull request by Adam-D-Lewis**
- K8s 1.25,
- JHub 3.1,
- Updated Nvidia drivers,
- Remove CDS Dashboards
- Enable Cost Allocation by default on GKE
- allow custom tags and labels to be added to GCP deploments

## Reference Issues or PRs
Resolves https://github.com/nebari-dev/nebari/issues/1849

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds a feature)
- [x] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

**Comment by Adam-D-Lewis** [comment]
This shouldn't be meged until we add some docs about CDS Dashboards deprecation.

**Comment by iameskild** [comment]
@Adam-D-Lewis if it's alright with you, I'm going to resolve the conflicts in this PR and get it merged.

**Comment by Adam-D-Lewis** [comment]
For sure, thanks Eskild!

File: src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ resource "helm_release" "jupyterhub" {
 
   repository = "https://jupyterhub.github.io/helm-chart/"
   chart      = "jupyterhub"
-  version    = "1.2.0"
+  version    = "2.0.0"
```
[the remark this answers is not in the document]
Comment (iameskild) [reply src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf:20]: Ah you're right! Thanks!

**Review by iameskild — APPROVED** [review]
I tested on an Azure deployment and this works if used with [these images](https://github.com/nebari-dev/nebari-docker-images/pull/85). The kubernetes test in CI doesn't use those images and so this fails. 

Thanks @Adam-D-Lewis 🎉

**Pull request merged**